foursight_core/react/api/gac.py

```diff
@@ -17,17 +19,28 @@
 class Gac:
 
     @staticmethod
-    def get_secrets_names() -> list:
+    def get_secret_names() -> list:
         try:
-            boto_secrets_manager = boto3.client('secretsmanager')
-            return [secrets['Name'] for secrets in boto_secrets_manager.list_secrets()['SecretList']]
+            secrets_manager = boto3.client("secretsmanager")
+            secret_names = []
+            pagination_token = None
+            response = secrets_manager.list_secrets(MaxResults=100)
+            while True:
```

Comment:
Can we abstract this into a generator function that does this so we don't have to have loops like this all over the place/

Reply:
Sounds good. Actually, I should probably update dcicutils.secret_utils._all_secrets (and also there, it is not taking pagination into account - I forget where but I think I did run into a situation where I needed it, i.e. where there where more the 100 results (though now that I say it that seems unlikely)).